art-community-platform/backend/app/api/helpers/recommendation_helpers.py

```diff
@@ -40,5 +40,66 @@ def get_tags_of_favourites(user):
     return tags
 
 
+#get 8 most popular users in the system
+def get_popular_users():
+    try:
+        user_list = User.objects.all()
+
+    except:
```

Comment:
Taking care of this kind of possible erroneous cases about database access is really important.
It also makes debugging easier.
I appreciate you!